persistence-model/src/main/scala/hmda/persistence/messages/commands/institutions/InstitutionCommands.scala

```diff
@@ -6,5 +6,5 @@ import hmda.persistence.messages.CommonMessages.Command
 object InstitutionCommands {
   case class CreateInstitution(i: Institution) extends Command
   case class ModifyInstitution(i: Institution) extends Command
-  case class GetInstitution(id: String) extends Command
+  case class GetInstitutionByRespondentId(id: String) extends Command
```

Comment:
I don't see the `protobuf` converter/serialization code for this message in this PR. It'd be nice to include it.

Reply:
Should be done now.